src/firebase-namespace.ts

```diff
@@ -169,14 +173,14 @@ export class FirebaseNamespaceInternals {
     delete this.apps_[appName];
   }
 
-  /*
+  /**
    * Registers a new service on this Firebase namespace.
    *
-   * @param {string} serviceName The name of the Firebase service to register.
-   * @param {FirebaseServiceFactory} createService A factory method to generate an instance of the Firebase service.
-   * @param {object} [serviceProperties] Optional properties to extend this Firebase namespace with.
-   * @param {AppHook} [appHook] Optional callback that handles app-related events like app creation and deletion.
-   * @return {FirebaseServiceNamespace<FirebaseServiceInterface>} The Firebase service's namespace.
+   * @param serviceName The name of the Firebase service to register.
+   * @param createService A factory method to generate an instance of the Firebase service.
+   * @param serviceProperties Optional properties to extend this Firebase namespace with.
+   * @param appHook Optional callback that handles app-related events like app creation and deletion.
```

Comment:
Do these need back-ticks? `createService`.
The original docs didn't have `` either so it might be okay :)

Reply:
The parameter names don't need backticks. We do need to do some additional formatting changes to meet some API Extractor requirements. I'll do them in a separate PR.